src/NBitcoin/NetworkOptions.cs

```diff
@@ -0,0 +1,149 @@
+﻿namespace NBitcoin
+{
+    /// <summary>
+    /// The current use-case for this class is to provide option-dependent serialization.
+    /// It is a drop-in replacement for the legacy TransactionOptions enumeration.
+    /// 
+    /// The NetworkOptions instance is intended to be relayed from the Network object to 
```

Comment:
Put the comments from this line and onward inside a <remarks>

Reply:
Not sure what is required here.